### PR TITLE
[READY] LatyshevEV lab1

### PR DESCRIPTION
**Pull request by JojoBroono**

### CODE REVIEW

**Comment by Toxa-man** [comment]
Everything is almost correct except some points
1) Instead of pusing only source and executable file, push the whole folder with project (it contains .sln file and so on). It's my inaccuracy in README and it will be fixed soon
2) Some remarks about sources

File: LatyshevEV/lab1/Code [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#include<string>
+using namespace std;
+int main()
+{
+	setlocale(0, "");
+	float left;
+	char sign;
+	float right;
+	bool flag = 0;
+	cout << "Левый операнд пожалуйсто" << endl;
```
Comment (Toxa-man) [inline LatyshevEV/lab1/Code]: Use more formalistic text style
We are not at "KVN" show
The same for output strings below

File: LatyshevEV/lab1/Code [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+			cout << "Глупенький, на ноль делить нельзя"<<endl; 
+		}
+		break;
+	default:cout << "Знак не валиден"<< endl;
+		flag = 1;
+		break;
+	}
+	if (flag == 0)
+	{
+		cout << result;
```
Comment (Toxa-man) [inline LatyshevEV/lab1/Code]: Make output result more human-readable, e.g. "2 + 3 = 5"

File: LatyshevEV/lab1/Code [hunk 3]
```diff
@@ -0,0 +1,48 @@
+#include<iostream>
+#include<string>
+using namespace std;
+int main()
+{
+	setlocale(0, "");
+	float left;
+	char sign;
+	float right;
+	bool flag = 0;
```
Comment (Toxa-man) [inline LatyshevEV/lab1/Code]: Use true and false keywords instead of numeric literals

**Comment by Toxa-man** [comment]
None of remarks were fixed

**Comment by JojoBroono** [comment]
All remarks were fixed, maybe i do smth wrong with GitHub?
cppcode

**Comment by Toxa-man** [comment]
> All remarks were fixed, maybe i do smth wrong with GitHub?
> cppcode

You pushed your changes to master branch, and current pull request was created from branch "LatyshevEV_lab1"
You need to move your changes to appropriate branch

**Pull request merged**